products/jira.md

```diff
@@ -152,6 +200,62 @@ releases:
     link:
       https://confluence.atlassian.com/jirasoftware/jira-software-8-13-x-release-notes-1018783360.html
 
+-   releaseCycle: "8.12"
+    releaseDate: 2020-08-25
+    eol: 2022-08-26
+    latest: "8.12.3"
+    latestReleaseDate: 2020-10-06
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-8-12-3-1026523933.html
+
+-   releaseCycle: "8.11"
+    releaseDate: 2020-07-14
+    eol: 2022-07-15
+    latest: "2020-08-16"
+    latestReleaseDate: 
```

Comment:
```suggestion
    latest: "8.11.1"
    latestReleaseDate: 2020-08-16
```

Reply:
Applied latest.py with the releases from https://github.com/endoflife-date/release-data/blob/main/releases/jira.json, this was the only error.